src/index.js

```diff
@@ -11,25 +11,23 @@ const NotificationStore = {
         closeOnClick: true,
         showClose: true
     },
-    setOptions (options) {
+    setOptions(options) {
         this.settings = {...this.settings, ...options}
     },
-    removeNotification (timestamp) {
+    removeNotification(timestamp) {
         const indexToDelete = this.state.findIndex(n => n.timestamp === timestamp)
-        if (indexToDelete !== -1) {
-            this.state.splice(indexToDelete, 1)
-        }
+        if (indexToDelete !== -1) this.state.splice(indexToDelete, 1)
```

Comment:
I guess that is not a good code pratice. Remove the braces can be tricky when read them.

Reply:
Hello, @amandiobm 
First of all, thank you for the great plugin
Secondly, I returned braces, but I think that in this case the code is read without difficulty